packages/playwright-core/src/server/fetch.ts

```diff
@@ -282,8 +282,15 @@ export abstract class APIRequestContext extends SdkObject {
           progress.log(`  ${name}: ${value}`);
 
         const cookies = this._parseSetCookieHeader(response.url || url.toString(), response.headers['set-cookie']) ;
-        if (cookies.length)
-          await this._addCookies(cookies);
+        if (cookies.length) {
+          try {
+            await this._addCookies(cookies);
+          } catch (e) {
+            // Cookie value is limited by 4096 characters in the browsers. If setCookies failed,
```

Comment:
What are browsers doing in terms of: 2 good cookies get returned and one bad cookie. Do they then drop the whole set of cookies or only ignore the one bad cookie?

Reply:
oh I see your change below, great!

Reply:
They ignore invalid cookies and accept others.